vms/platformvm/utxo/camino_handler_test.go

```diff
@@ -106,3 +109,421 @@ func TestVerifySpendUTXOsWithLocked(t *testing.T) {
+						credentials[0].Sigs[0],
+					},
+				},
+			},
+			signers: []verify.State{
+				&secp256k1fx.TransferOutput{
+					Amt: 1,
+					OutputOwners: secp256k1fx.OutputOwners{
+						Addrs: []ids.ShortID{
+							address0,
+						}, Threshold: 1,
+					},
+				},
+			},
+			producedAmounts: map[ids.ID]uint64{
+				h.ctx.AVAXAssetID: 1,
+			},
+			expectedErr: ErrWrongUTXONumber,
+		},
+		"Wrong UTXO AssetID, One Input, No Outputs": {
+			utxos: []*avax.UTXO{{
+				Asset: avax.Asset{ID: ids.GenerateTestID()},
+				Out: &secp256k1fx.TransferOutput{
+					Amt: 1,
+				},
+			}},
+			ins: []*avax.TransferableInput{{
+				Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+				In: &secp256k1fx.TransferInput{
+					Amt: 1,
+				},
+			}},
+			outs: []*avax.TransferableOutput{},
+			creds: []verify.Verifiable{
+				&secp256k1fx.Credential{},
+			},
+			signers:         []verify.State{},
+			producedAmounts: map[ids.ID]uint64{},
+			expectedErr:     errAssetIDMismatch,
+		},
+		"Zero Signers, Threshold=1": {
+			utxos: []*avax.UTXO{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			ins: []*avax.TransferableInput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					In: &secp256k1fx.TransferInput{
+						Amt: 1,
+					},
+				},
+			},
+			outs: []*avax.TransferableOutput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			creds: []verify.Verifiable{
+				&secp256k1fx.Credential{},
+			},
+			signers: []verify.State{
+				&secp256k1fx.TransferOutput{
+					Amt: 1,
+					OutputOwners: secp256k1fx.OutputOwners{
+						Addrs: []ids.ShortID{
+							address0,
+						}, Threshold: 1,
+					},
+				},
+			},
+			producedAmounts: map[ids.ID]uint64{},
+			expectedErr:     secp256k1fx.ErrTooFewSigners,
+		},
+		"One Signer, Threshold=1": {
+			utxos: []*avax.UTXO{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			ins: []*avax.TransferableInput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					In: &secp256k1fx.TransferInput{
+						Amt: 1,
+						Input: secp256k1fx.Input{
+							SigIndices: []uint32{0},
+						},
+					},
+				},
+			},
+			outs: []*avax.TransferableOutput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			creds: []verify.Verifiable{
+				&secp256k1fx.Credential{
+					Sigs: [][crypto.SECP256K1RSigLen]byte{
+						credentials[0].Sigs[0],
+					},
+				},
+			},
+			signers: []verify.State{
+				&secp256k1fx.TransferOutput{
+					Amt: 1,
+					OutputOwners: secp256k1fx.OutputOwners{
+						Addrs: []ids.ShortID{
+							address0,
+						}, Threshold: 1,
+					},
+				},
+			},
+			producedAmounts: map[ids.ID]uint64{},
+			expectedErr:     nil,
+		},
+		"One Signer, Threshold=2": {
```

Comment:
It says `Threshold=2` but is `Threshold=1`
But more interesting test would be to make it work, maybe `sigIndices` are wrong.
Let's make this test work without error, as it's perfect example of MSig.

So it should be:
- Signers: `Threshold=1, 2 Addresses (make sure they are sorted address1 < address2 bytewise`
- Creds `One signature as Threshold=1`
- SigIndices should correctly point which of 2 addresses signed
- It should run without errors